ob-pattern-1/templates/obam/obam-server-service.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@kdanW update the license header year.

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/2b2f6f4c2e0c8feada3a3eda00548a132db73a26